api.planx.uk/modules/gis/service/index.js

```diff
@@ -22,8 +20,7 @@ const localAuthorities = { digitalLand, braintree, scotland };
  *        description: Well-Known Text (WKT) formatted polygon or point
  */
 export async function locationSearch(req, res, next) {
-  // 'geom' param signals this localAuthority has data available via DigitalLand, "ready" teams are configured in PlanningConstraints component in editor
-  //   swagger doc intentionally doesn't cover legacy custom GIS hookups for Braintree and Scotland demo
+  // 'geom' param signals this localAuthority has data available via Planning Data
```

Comment:
Aha this answers my previous question - maybe a check against `hasPlanningData` in both places is a bit more robust / expected.